rackcity/views/user_views.py

```diff
@@ -239,6 +240,10 @@ def user_delete(request):
             status=HTTPStatus.BAD_REQUEST,
         )
     else:
+        deleted_user_assets = Asset.objects.filter(owner=username)
+        for asset in deleted_user_assets:
+            asset.owner = None
+            asset.save()
```

Comment:
after deleting the user, iterate over any assets who had the deleted user's username, set its owner to null